appeals/web/src/server/lib/mappers/components/folder.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Returns the instructions for a costs folder table item
+ *
+ * @param {Object} options
+ * @param {string} options.id
+ * @param {string} options.text
+ * @param {string} [options.statusText]
+ * @param {string} options.link
+ * @param {import('@pins/appeals.api').Appeals.FolderInfo|null|undefined} options.folderInfo
+ * @returns {Instructions}
+ */
+export function costsFolderTableItem({ id, text, statusText = 'Received', link, folderInfo }) {
```

Comment:
This is a nice improvement, and this is quite a pedantic point, but I think the name of this function could be tweaked to `costsFolderRowTableItem` just to clarify this is for a folder row, not the folder itself (I initially thought this must be for the manage folder page, which also uses a table and is more aligned to what I think of as a "folder", but I get that it's very subjective - in my opinion just adding "row" to the name helps a lot).